transform/transformers/pck_transformer.py

```diff
@@ -12,9 +12,13 @@ class PCKTransformer:
     comments_questions = ['147', '146a', '146b', '146c', '146d', '146e', '146f', '146g', '146h', '146i', '146j', '146k']
     rsi_turnover_questions = ["20", "21", "22", "23", "24", "25", "26"]
     rsi_currency_questions = rsi_turnover_questions + ["27"]
-    qcas_currency_questions = ['681', '688', '689', '695', '696', '697', '703', '704', '707', '708', '709', '710', '711', '712']
     employee_questions = ["50", "51", "52", "53", "54"]
 
+    qcas_acquisitions_questions = ['688', '695', '703', '707', '709', '711']
+    qcas_disposals_questions = ['689', '696', '704', '708', '710', '712']
+    qcas_calculated_total = ['692', '693']  # Calculated summary values
```

Comment:
Jimi is saying these should be `714` and `715` respectively. Will get confirmation before we merge. Otherwise this looks good.

Reply:
Seems we may have need for `692`, `693`, `714` and `715`. Jimi will need to define the need properly. By all accounts `693` and `715` are one and the same.